docs/jwst/extract_2d/main.rst

```diff
@@ -78,6 +78,80 @@ of ``GrismObjects`` outside of these, the ``GrismObject`` itself can be imported
 The dispersion direction will be documented by copying keyword "DISPAXIS"
 (1 = horizontal, 2 = vertical) from the input file to the output cutout.
 
+Following are examples of how to customize the list of grism objects used in extract_2d.
+The input file must have a WCS assigned to it by running ``assign_wcs``. The default values
+for  wavelength range and the spectral orders are stored in the ``wavelengthrange``
+reference file which can be retrieved from CRDS. A user may supply a different
+wavelength range by passing `None` to ``reference_files``. In this case the spectral
+orders to be extracted and their corresponding wavelength range will be taken
+from the ``wavelength_range`` parameter.
```

Comment:
How does a user specify both wavelength ranges AND spectral orders via a single parameter? Or is that explained later?

Reply:
The `wavelength_range` parameter is a dictionary of 
`{spectral_order: (lam_min, lam_max)}`